morpheus/_lib/include/morpheus/stages/doca_source_kernels.hpp

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Can this file be moved to `morpheus/doca` instead of in the stages directory?

Reply:
This file is specific to the doca source stage. I don't see a more appropriate place for it. Perhaps reorganizing the code in the `stages` directory to allow for more complex stage implementations makes sense, but moving this code in to the more reusable `doca` directory just obfuscates what the code is used for in my opinion.